pkg/middleware/middleware.go

```diff
@@ -59,8 +60,22 @@ func Process(ctx *quaycontext.QuayRegistryContext, quay *v1.QuayRegistry, obj cl
 			return dep, nil
 		}
 
+		// Set env variable overrides
+		var envVars map[string]string
+		switch quayComponentLabel {
+		case "postgres":
+			envVars = getEnvOverrideForComponent(quay, v1.ComponentPostgres)
+		case "clair-postgres":
+			envVars = getEnvOverrideForComponent(quay, v1.ComponentClair)
+		case "objectstorage":
+			envVars = getEnvOverrideForComponent(quay, v1.ComponentObjectStorage)
+		}
+		for key, val := range envVars {
+			dep.Spec.Template.Spec.Containers[0].Env = append(dep.Spec.Template.Spec.Containers[0].Env, corev1.EnvVar{Name: key, Value: val})
```

Comment:
This will set only in the first container. Is this expected? Why not set in all of them?